content/800-data-platform/250-environments/400-delete.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: 'Delete an environment'
+metaTitle: 'Delete a project environment in the Prisma Data Platform'
+metaDescription: 'If you no longer need an environment in your project or want to reduce the number of environments to the quota in the Scale plan, you can delete one of the project environments.'
+tocDepth: 3
+---
+
+<TopBlock>
+
+If you no longer need an environment in your project or want to reduce the number of environments to the quota in the Scale plan, you can delete one of the project environments.
+
+<Admonition type="alert">
+
+**Important**<br /><br />
+
+You cannot delete the default environment.<br /><br />
+
+To delete the current default environment, you must first [make one of the other environments to be the default](/data-platform/environments/edit-settings/change-default-environment).
```

Comment:
```suggestion
To delete the current default environment, you must first [make one of the other environments the default](/data-platform/environments/edit-settings/change-default-environment).
```

Also, I think this can be a second sentence in the preceding paragraph.